export/export_handler_test.go

```diff
@@ -150,3 +151,45 @@ func TestTouch(t *testing.T) {
 		t.Errorf("expected %s, got %s\n", testData, string(val))
 	}
 }
+
+func Test_getGenerateSuccess(t *testing.T) {
+
+	tests := []struct {
+		name                   string
+		lastKeysInPrefixRegexp []string
+		expectTrue             []string
+		expectFalse            []string
+		want                   func(path string) bool
+	}{
+		{
+			name:                   "one regex",
+			lastKeysInPrefixRegexp: []string{".*\\.success$"},
+			expectTrue:             []string{"a.success", "other.success"},
+			expectFalse:            []string{"dfd", "a.suc", "a.successer"},
+			want:                   nil,
+		},
+		{
+			name:                   "two regexes",
```

Comment:
```suggestion
			name:                   "two regexps",
```
(one advantage of "regexp" is that you don't have smart alecs suggesting "regexen" as the plural.